Unified/closor.py

```diff
@@ -317,6 +319,34 @@ def rank( wfn ):
 %s 
 This is an automated message.
 """%( bname, 
+      bname,
+      issues)
+            to = ['hn-cms-relval@cern.ch']
+            sendEmail(subject, text, destination=to )
+            ## just announced ; take it out now.
+            #BI.pop( bname )
+            subject = "Release Validation low statistics"
```

Comment:
Let's put `Low statistics` first in case the email subject gets truncated on phone.

Reply:
Changed subject so that even it gets truncated on phone, you will see ``Low Statistics``